src/controller/matlab/mgenerate.py

```diff
@@ -517,28 +518,28 @@ def main(args=None):
                         "supervisor")  # DEPRECATED
     generator.gen(FUNC, "wb_supervisor_node_enable_pose_tracking(sampling_period, node, from_node)", "supervisor")
     generator.gen(FUNC, "wb_supervisor_node_export_string(noderef)", "supervisor")
+    generator.gen(FUNC, "wb_supervisor_node_get_base_node_field(noderef, fieldname)", "supervisor")
+    generator.gen(FUNC, "wb_supervisor_node_get_base_node_field_by_index(noderef, fieldindex)", "supervisor")
     generator.gen(FUNC, "wb_supervisor_node_get_base_type_name(noderef)", "supervisor")
     # generator.gen(FUNC, "wb_supervisor_node_get_center_of_mass(noderef)", "supervisor")
     # generator.gen(FUNC, "wb_supervisor_node_get_contact_point(noderef, index)", "supervisor")
     generator.gen(FUNC, "wb_supervisor_node_get_contact_point_node(noderef, index)", "supervisor")
     # generator.gen(PROC, "wb_supervisor_node_get_contact_points(noderef, include_descendants)", "supervisor")
     generator.gen(FUNC, "wb_supervisor_node_get_def(noderef)", "supervisor")
     generator.gen(FUNC, "wb_supervisor_node_get_field(noderef, fieldname)", "supervisor")
-    generator.gen(FUNC, "wb_supervisor_node_get_field_by_index(noderef, fieldname)", "supervisor")
+    generator.gen(FUNC, "wb_supervisor_node_get_field_by_index(noderef, fieldindex)", "supervisor")
```

Comment:
Is this just a costmetic fix or does it change fix an actual bug and/or affect existing code? If it's not just costmetic, it should probably be added to the changelog.

Reply:
Afaik, this is just cosmetic.